CMakeLists.txt

```diff
@@ -11,10 +11,20 @@ option(COMPILE_TESTS "Compile the tests" OFF)
 
 add_library(libblisp_obj OBJECT
         lib/blisp.c
+        lib/blisp_easy.c
+        lib/blisp_util.c
         lib/chip/blisp_chip_bl60x.c
-        lib/chip/blisp_chip_bl70x.c lib/blisp_easy.c)
+        lib/chip/blisp_chip_bl70x.c)
 
 target_include_directories(libblisp_obj PRIVATE ${CMAKE_SOURCE_DIR}/include/)
+if (NOT CMAKE_C_COMPILER_ID MATCHES "MSVC")
```

Comment:
I'm guessing this isn't working with the build failures?

Reply:
So, for the first one, `float` to `uint32_t`, what is missing is an explicit conversion. I will add it as a separate commit.  The other two, are about the `uint16_t payload_size` parameter of `blisp_send_command`.

Reply:
OK. I'm done :)
If you think it's OK to merge with the two warnings only MSVC raises, I'm OK too.

I can work on these warnings in the near future. I promise it won't take me 2 years to come back to it (check the age of this PR :sweat_smile:).

Reply:
I'm also to blame for the age, no one was really monitoring PR's well and I forgot about it 😓 

I think its good to go indeeed